.github/workflows/tests.yaml

```diff
@@ -83,8 +82,10 @@ jobs:
         run: |
           brew update
           brew install multipass
-          multipass version
           sleep 20
+          multipass set local.driver=qemu
```

Comment:
I thought this was fixed with the latest multipass?

Reply:
Ah yes, thanks for the reminder

Reply:
Well, apparently it's not!  I wonder why this is working on `main`...

https://github.com/canonical/multipass/releases/tag/v1.12.0

https://github.com/canonical/craft-providers/actions/runs/5392910938/jobs/9791898806?pr=338#step:5:1187

Reply:
pakistani_cricket_fan_disappointed.gif